gguf-py/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "gguf"
-version = "0.4.6"
+version = "0.4.7"
```

Comment:
Shouldn't this be a minor version bump since we are adding GGUFReader?

Reply:
Sure, do you want to change it or should I? (I assume you mean it should be `0.5,0`?)

Reply:
I changed it.